code/modules/clothing/shoes/clown.dm

```diff
@@ -53,3 +53,9 @@
 	desc = "The adorable sound they make when you walk will mean making friends is more likely."
 	icon_state = "meown_shoes"
 	squeak_sound = list('sound/effects/footstep/meowstep1.ogg'=1) //mew mew mew mew
+
+/obj/item/clothing/shoes/clown_shoes/moffers
+	name = "moffers"
+	desc = "No moths died in the making of these slippers."
```

Comment:
```suggestion
	desc = "The question on whether moths were harmed in the making of these cannot be answered."
```

Reply:
is moffers supposed to be a play on words for something btw

Reply:
It's a silly way to say moth slippers

Reply:
Description changed to include "were harmed" instead of "died"